zk/stages/stage_batches.go

```diff
@@ -30,7 +30,7 @@ import (
 )
 
 const (
-	STAGE_PROGRESS_SAVE    = 3000000
+	STAGE_PROGRESS_SAVE    = 100_000
```

Comment:
I changed it to a lower number because now the stage will simply exit if there is a network blip which could lose you a lot of progress in the downloading